daemon/cmd/daemon.go

```diff
@@ -570,12 +569,13 @@ func newDaemon(ctx context.Context, cleaner *daemonCleanup, params *daemonParams
 				d.restoredCIDRs = append(d.restoredCIDRs, k.Prefix())
 				oldNIDs = append(oldNIDs, nid)
 			} else if nid == identity.ReservedIdentityIngress && v.TunnelEndpoint.IsZero() {
-				oldIngressIPs = append(oldIngressIPs, k.IPNet())
-				ip := k.IPNet().IP
```

Comment:
One other thing that I just noticed: these are the only remaining call sites of `ipcachemap.Key.IPNet`, so it can probably be removed too as part of this PR or in a follow-up.

Reply:
`IPNet()` is already removed in this PR.